src/bench/block_assemble.cpp

```diff
@@ -41,7 +41,8 @@ static CTxIn MineBlock(const CScript& coinbase_scriptPubKey)
     auto block = PrepareBlock(coinbase_scriptPubKey);
 
     while (!CheckProofOfWork(block->GetHash(), block->nBits, Params().GetConsensus())) {
-        assert(++block->nNonce);
+        block->nNonce++;
```

Comment:
nit: could be `++block->nNonce` as before

Reply:
@Empact Updated. Please re-review :-)